package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "webpack": "NODE_ENV=production webpack -p --bail",
     "watch": "webpack --watch",
+    "prestart": "npm install",
```

Comment:
I don't see `npm run prestart` anywhere, and if this is meant to be run by the user, can't they just run `npm install` directly?